index.html

```diff
@@ -84,10 +84,23 @@ <h3 class="portfolio-header" id="portfolio-section">Portfolio</h3>
 		</div>
 	</main>
 	
-	<!-- <div class="contact" id="contact-section">
+	<div class="contact" id="contact-section">
 		<h3>Contact</h3>
 		<p>Feel free to drop me a message via email or on my social media.</p>
+		<div class="email">
+			<ion-icon name="mail"></ion-icon>
+			<p>daniellaskinner120@gmail.com</p>
+		</div>
+		<div class="social-buttons">
+			<ion-icon name="logo-github"></ion-icon>
+			<ion-icon name="logo-twitter"></ion-icon>
+			<ion-icon name="logo-facebook"></ion-icon>
+			<ion-icon name="logo-instagram"></ion-icon>
+			<ion-icon name="logo-linkedin"></ion-icon>
+		</div>
```

Comment:
As mentioned, might be a good idea to add an <a> tag to link to the relevant social sites